prclz/reblock/planar_graph.py

```diff
@@ -0,0 +1,1204 @@
+import argparse
```

Comment:
is this file mostly identical to the one in the prclz folder in the old repo? if so, i think it might make sense to consolidate them

Reply:
Hmm, I don't think so actually bc at the very least I implemented this in igraph and it diverged a while ago. I'm not sure what the one in prclz is really trying to do but I think they're pretty different at this point.